integreat_cms/cms/views/regions/region_actions.py

```diff
@@ -51,7 +51,7 @@ def delete_region(
         messages.error(
             request,
             format_html(
-                "{}<ul class='list-disc pl-4'>{}</ul>",
```

Comment:
It probably makes sense to undo this change, since it is unrelated.
I am in favor of enforcing a consistent order of attributes, but that should probably be done in another pr for the whole repository.